packages/lesswrong/components/recommendations/RecommendationsAndCurated.tsx

```diff
@@ -123,7 +124,7 @@ const RecommendationsAndCurated = ({
     const renderContinueReading = currentUser && (continueReading?.length > 0) && !settings.hideContinueReading
 
     return <SingleColumnSection className={classes.section}>
-      {<AnalyticsContext pageSectionContext="gatherTownWelcome">
+      {!isMobile() && <AnalyticsContext pageSectionContext="gatherTownWelcome">
```

Comment:
What does this do during SSR? Did you test that this doesn't break SSR in some weird ways?

Reply:
I in fact was not sure this even worked, and was wondering about the best way to actually accomplish this. I was hoping to avoid rendering it at all on mobile so we didn't have to run any queries or anything. If that's not actually possible I can just hide it with jss.

Reply:
Yeah, I would just hide it in jss.

Reply:
I find that I do look at who is online on mobile when I check the site. My solution here is to just hide the list of events on mobile.